sdk/src/main/res/values/strings.xml

```diff
@@ -61,6 +59,29 @@
     <string name="payment_chooser_title">Payment Methods</string>
     <string name="internet_banking_chooser_title">Internet Banking</string>
     <string name="installments_title">Installments</string>
-    <string name="econtext_title">EContext</string>
+    <string name="econtext_title">econtext</string>
```

Comment:
That use all lowercase?

Reply:
Yeah, it seems to be like that according to our copy writer.